src/Engine/filelist.cmake

```diff
@@ -72,13 +73,15 @@ set(engine_headers
     Data/ShaderProgram.hpp
     Data/ShaderProgramManager.hpp
     Data/SimpleMaterial.hpp
+    Data/SynchronizableObject.hpp
     Data/Texture.hpp
     Data/TextureManager.hpp
     Data/ViewingParameters.hpp
     Data/VolumeObject.hpp
     Data/VolumetricMaterial.hpp
     FrameInfo.hpp
     OpenGL.hpp
+    pch.hpp
```

Comment:
This file should not be mentioned here. If `RADIUM_ENABLE_PCH` option is `OFF`, this file will not be parsed/compiled.

Reply:
Might come from generateFilelsitForModule.sh, PR #988 proposes a fix.